cribl_stream/README.md

```diff
@@ -0,0 +1,70 @@
+# Cribl Stream
+
+## Overview
+[Cribl Stream][1] helps you process machine data – logs, instrumentation data, application data, metrics, and so on – in real time, and deliver it to your analysis platform of choice. It allows you to:
+
+- Add context to your data, by enriching it with information from external data sources.
+- Help secure your data, by redacting, obfuscating, or encrypting sensitive fields.
+- Optimize your data, per your performance and cost requirements.
+
+Visualize your Cribl Stream metrics in Datadog with the out-of-the-box dashboard.
+
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][5] for a list of metrics provided by this integration.
+
+### Events
+
+The Cribl Stream integration does not include any events.
+
+### Service Checks
+
+The Cribl Stream integration does not include any service checks.
+
+### Dashboards
+
+Use the out-of-the-box dashboard to view the performance of the Stream with base metrics like events per second, bytes per second, input types, output types, and infrastructure metrics. Monitor reduction percentages by events or bytes, which is useful for improving search performance or licensing and infrastructure costs for the systems of analysis.
+
+## Setup
+You can send your Cribl Stream [internal metrics][6] to the Datadog API, allowing you to monitor your applications and infrastructure without the Datadog Agent. 
+
+### Installation
+
+#### Datadog
+Navigate to _API Keys_ under _Organization Settings_ and create an API Key for Cribl to send data.
+
+#### Cribl
+1. In Cribl, navigate to _Quick Connects_ and click the _+Add Source_ button. 
+![step1](/images/cribl_dd_1.png)
+2. Scroll down to _System Internal_ , hover over _Cribl Internal_ and choose _Select Existing_. Enable both _CriblLogs_ and _CriblMetrics_.  
+    - **Note**: Both sources must have **Quick Connect** enabled instead of **Routes**.
+![step3](/images/cribl_dd_3.png)
+
+3. Click the _+Add Destination_ button.
+4. Scroll to the _Datadog_ tile and click _+Add New_.
+5. Give a name to the input (for example, Cribl_Datadog).
+![step4](/images/cribl_dd_4.png)
+
+6. Next, enter your _Datadog API Key_ and make sure your site is either _US_ or _Europe_  
+7. Add any Datadog tags, a Message Field, Source, or Host information. For more information, see the [Cribl Datadog Destination documentation][3].
+8. Click _Save_.
+9. Select _Passthru_ to connect Cribl Metrics to your Datadog destination.
+![step5](/images/cribl_dd_6.png)
+
+![complete](/images/cribl_dd_5.png)
+
+## Uninstallation
+Use the [delete dashboard][4] option within the Cribl Stream dashboard settings to delete the Cribl Stream dashboard.
+
+## Support
+Information about how and where to go for support for this integration.
```

Comment:
Can you add support contacts from Cribl here, and a link to any relevant documentation?

Reply:
Yes, I added them to the page.